Microsoft.Toolkit.Uwp.UI.Controls/SlidableListItem/SlidableListItem.cs

```diff
@@ -577,5 +631,67 @@ public object RightCommandParameter
                 SetValue(RightCommandParameterProperty, value);
             }
         }
+
+        /// <summary>
+        /// Gets the SwipeStatus for current swipe status
+        /// </summary>
+        public SwipeStatus SwipeStatus
+        {
+            get
+            {
+                return (SwipeStatus)GetValue(SwipeStatusProperty);
+            }
+
+            private set
+            {
+                SetValue(SwipeStatusProperty, value);
+            }
+        }
+    }
+
+    /// <summary>
+    /// Types of swipe status.
+    /// </summary>
+    public enum SwipeStatus
+    {
+        /// <summary>
+        /// Swiping is not occuring.
+        /// </summary>
+        NotRunning,
```

Comment:
My vote is for Idle (will have that state both before and after swiping). But I’m not native English speaking so I wait for your advice :). Will make these changes tomorrow.